### PR TITLE
Update readme to match the design

### PR DESCRIPTION
**Pull request by Gela**
# Description
 

Update readme to match the sample design "How to use".

## Type of change

- [ ] Bug fix
- [ ] New sample implementation
- [ ] Sample viewer enhancement
- [x] Other enhancement

### CODE REVIEW

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/ApplyUniqueValuesWithAlternateSymbols/README.md [hunk 1]
```diff
@@ -16,23 +16,22 @@ Zoom in and out of the map to see alternate symbols at each scale. To go back to
...
 
-1. Create a feature layer using the service url and add it to the list of operational layers.
-2. Create two alternate symbols (a blue square and a yellow diamond) to be used for the unique value. To create an alternate symbol:
     a. Create a symbol using `SimpleMarkerSymbol`.
-    b. Convert the Simple Marker Symbol to a Multilayer Symbol.   
-    c. Set the valid scale range through reference properties on the multilayer point symbols blue square and yellow diamond by calling `SymbolReferenceProperties`.
     
-3. Create a third multilayer symbol to be used when creating the unique value:
-    a. Create a red triangle and set the reference properties.
+    b. Convert the simple marker symbol to a `MultilayerSymbol` using `SimpleMarkerSymbol::toMultilayerSymbol`.
```
Comment (AndrewBladon) [inline ArcGISRuntimeSDKQt_QMLSamples/Layers/ApplyUniqueValuesWithAlternateSymbols/README.md:24]: Should `SimpleMarkerSymbol::toMultilayerSymbol` be changed to something like `SimpleMarkerSymbol.toMultilayerSymbol()` to make it look more like Qml?

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/ApplyUniqueValuesWithAlternateSymbols/README.md [hunk 2]
```diff
@@ -16,23 +16,22 @@ Zoom in and out of the map to see alternate symbols at each scale. To go back to
...
     
-4. Create a unique value using the red triangle from step three and the list of alternate symbols from step two.
-5. Create a unique value renderer and add the unique value from step four to it.
+    c. Set the valid scale range through reference properties on the multilayer point symbols blue square and yellow diamond by calling `MultilayerSymbol::setReferenceProperties(new SymbolReferenceProperties(minScale, maxScale));`.
```
Comment (AndrewBladon) [inline ArcGISRuntimeSDKQt_QMLSamples/Layers/ApplyUniqueValuesWithAlternateSymbols/README.md:26]: As above, should `MultilayerSymbol::setReferenceProperties(new SymbolReferenceProperties(minScale, maxScale));` be changed to something like `MultilayerSymbol.referenceProperties` to make it look more like Qml?

**Review by AndrewBladon — APPROVED** [review]
Looks good @Gela!

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/ApplyUniqueValuesWithAlternateSymbols/README.md (hunk 1 above)
Comment (Gela) [reply ArcGISRuntimeSDKQt_QMLSamples/Layers/ApplyUniqueValuesWithAlternateSymbols/README.md:24]: Yes I agree, I will update it

**Pull request merged**